pom.xml

```diff
@@ -2,7 +2,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>fi.hsl</groupId>
     <artifactId>transitdata-common</artifactId>
-    <version>1.0.9</version>
+    <version>1.0.12</version>
```

Comment:
Let's not bump the version in feature branches but when releasing (merging develop to master). Of course locally you can set any version you like. FYI the current version of common is 1.2.0.